server/controllers/clients/utils.js

```diff
@@ -0,0 +1,9 @@
+export const phases = {
+    PHASE_1: 'Phase 1: Information Gathering',
+    PHASE_2: 'Phase 2: Structure & Design',
+    PHASE_3: 'Phase 3: Initial Development',
+    PHASE_4: 'Phase 4: Testing/QA',
+    PHASE_5: 'Phase 5: Post Launch Maintenance',
+    PHASE_6: 'Phase 6: New Version Development',
+    PHASE_7: 'Phase 7: Terminated'
```

Comment:
Terminated or Completed ? Termination makes me feel like we deleted it

Reply:
I always saw Terminated as end of support, regardless of the reason. We can certainly rename the label to something everyone feels more comfortable having